### ISSUE TITLE
[Test Failure][Accessibility] String ‘Default package management format:’ is covered by its drop down in the Package Manager Settings window

### ISSUE THREAD

**Issue by v-luzh** [NuGet/Home#4503]
**NuGet Version:** Dev 2265 VS15 Localized NuGet Client Vsix
**VS Version:** D15Rel 26203.00 (select all languages and workloads)
**OS Version:** Windows 10 RS1_Release 14393.821.170127-1801

**Repro Steps:**
1.  Select **PTB** through ‘Tools -> Options -> International Settings’.
2.  Create a new C# Console Application via VS.
3.  Right-click on the project and click ‘Manage NuGet Packages…’.
4.  Click the settings button on the right-top of PM UI (or click Tools->**Package Manager Settings**).
5.  Click the **‘General’** tab.

**Expected Result:**
String ‘Default package management format:’ is not covered by its drop down.

**Actual Result:**
String ‘Default package management format:’ is covered by its drop down as below screenshot.
![stringcoveredbydropdown](https://cloud.githubusercontent.com/assets/20632625/22618316/59dd0752-eb14-11e6-9327-7f9fe4903443.png)

**Note:**
1. It repro on PTB,RUS,ESN,FRA,ITA.
2. It don’t repro on TRK,KOR,CHS,CHT,JPN,DEU,PLK,CSY.

**Comment by mishra14** [comment]
This is an accessibility bug.

**Comment by v-luzh** [comment]
Fixed on D15PreRel 26626.02+NuGut Client 4.3.0.4226. Close this bug now, thanks.
